frontend/src/scenes/funnels/funnelLogic.test.ts

```diff
@@ -183,4 +188,90 @@ describe('funnelLogic', () => {
             })
         )
     })
+
+    describe('as dashboard item', () => {
+        describe('props with filters and cached results', () => {
+            initKeaTestLogic({
+                logic: funnelLogic,
+                props: {
+                    dashboardItemId: 123,
+                    cachedResults: ['cached result'],
+                    filters: {
+                        events: [{ id: 2 }, { id: 3 }],
+                        properties: [{ value: 'lol', operator: PropertyOperator.Exact, key: 'lol', type: 'lol' }],
```

Comment:
I love learning what text people use to stand in for "any string here" 🚀

Reply:
lol